mode/examples/Demos/Graphics/Yellowtail/Yellowtail.pyde

```diff
@@ -16,20 +16,16 @@
 from gesture import Gesture
 
 nGestures = 36    # Number of gestures
-gestureArray = []
-currentGestureID = -1
-
-minMove = 3         # Minimum travel for a point
-tmpXp = []
-tmpYp = []
+minMove = 3       # Minimum travel for a point
 
 
 def setup():
+    global gestureArray, currentGestureID
     size(1024, 768, P2D)
     background(0, 0, 0)
     noStroke()
-    for _ in range(nGestures):
-        gestureArray.append(Gesture(width, height))
+    gestureArray = [Gesture(width, height) for _ in range(nGestures)]
```

Comment:
Can you comment further on that? Is it an aesthetic judgement, or can you articulate an heuristic?